packages/components/src/components/flyout/flyout-styles.ts

```diff
@@ -75,11 +79,26 @@ export const getComponentCss = (
       maxWidth: 'var(--p-flyout-max-width, 1180px)',
       color: primaryColor, // enables color inheritance for slotted content
       background: backgroundColor,
-      opacity: isOpen ? 1 : 0,
-      transform: isOpen ? 'initial' : `translate3d(${isPositionStart ? '-100%' : '100%'}, 0, 0)`,
-      transition: `opacity ${flyoutTransitionDuration} ${flyoutTransitionTimingFunction} ${
-        isOpen ? '0s' : flyoutTransitionDuration
```

Comment:
Didn't the `0s` get lost? Might be the reason for needed timeouts in e2e tests.

Reply:
the default value for `transition-delay` is `0s`